htmresearch/frameworks/sensorimotor/sensorimotor_experiment_runner.py

```diff
@@ -28,17 +28,21 @@
 from nupic.research.monitor_mixin.temporal_memory_monitor_mixin import (
   TemporalMemoryMonitorMixin)
 
-from sensorimotor.general_temporal_memory import GeneralTemporalMemory
-# Uncomment the line below to use GeneralTemporalMemory
-# from sensorimotor.general_temporal_memory import GeneralTemporalMemory
+# Uncomment the line below to use ExtendedTemporalMemory
+# from sensorimotor.extended_temporal_memory import ExtendedTemporalMemory
 from sensorimotor.temporal_pooler import TemporalPooler
 # Uncomment the line below to use SpatialTemporalPooler
 # from sensorimotor.spatial_temporal_pooler import SpatialTemporalPooler as TemporalPooler
 from sensorimotor.temporal_pooler_monitor_mixin import (
   TemporalPoolerMonitorMixin)
-class MonitoredGeneralTemporalMemory(TemporalMemoryMonitorMixin,
-                                     GeneralTemporalMemory): pass
-class MonitoredTemporalPooler(TemporalPoolerMonitorMixin, TemporalPooler): pass
+
+
+class MonitoredExtendedTemporalMemory(TemporalMemoryMonitorMixin, ExtendedTemporalMemory):
```

Comment:
I'd expect this to cause a Python error since `ExtendedTemporalMemory` hasn't been imported.